content/en/releases/patch-releases.md

```diff
@@ -78,9 +78,10 @@ releases may also occur in between these.
 
 | Monthly Patch Release | Cherry Pick Deadline | Target date |
 | --------------------- | -------------------- | ----------- |
-| February 2023         | 2023-02-10           | 2023-02-15  |
 | March 2023            | 2023-03-10           | 2023-03-15  |
 | April 2023            | 2023-04-07           | 2023-04-12  |
+| May 2023              | 2023-05-12           | 2023-05-17  |
```

Comment:
There's more than one month between April and May patch releases. We could release May patch releases one week earlier, but:

- there'll be low activity week prior and after KubeCon
- there are public holidays around 1st May in some countries

Should we leave it for May 17 or move it to May 10?

Reply:
I think going for May 17 is totally fine :+1: